packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -887,3 +889,40 @@ class HeroController extends NavigatorObserver {
     return toHero.child;
   };
 }
+
+/// Enables or disables [Hero]es in the widget subtree.
```

Comment:
This should document what it means that Heros are enabled/disabled. e.g. if it is disabled all Hero widgets in this subtree will not be considered for Hero animations or something like that.

Reply:
@najeira Try this.  Make sure to break any lines that exceed the 80 character width limit.


Enables or disables [Hero]es in the widget subtree.

When [enabled] is false, all Hero widgets in this subtree will not be considered for Hero animations.

When [enabled] is true (the default), Hero widgets may be involved in a Hero animation, like normal.